src/data/nat/cast/defs.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2014 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Gabriel Ebner
 -/
-import algebra.group.basic
+import algebra.group.defs
 import algebra.ne_zero
```

Comment:
Following the principal that `defs` files should try to only import other `defs` files, rather than files full of theory.